### PR TITLE
Ok/new dialogs

### PR DESCRIPTION
**Pull request by isntlazy**
Merging new dialogs to abstrac logic branch

### CODE REVIEW

File: src/components/TimelineWrapper.vue [hunk 1]
```diff
@@ -1054,7 +1078,9 @@ export default {
             filePath: item.filePath,
             timer_mark: this.$store.state.timer,
           };
-          await this.openEditorModal(data);
+          // await this.openEditorModal(data);
```
Comment (dacoaster) [inline src/components/TimelineWrapper.vue:1081]: @isntlazy - Please remove unused lines.

File: src/components/TimelineWrapper.vue [hunk 2]
```diff
@@ -1108,8 +1135,13 @@ export default {
...
     async handleActiveSession(id) {
-      this.activeSession = await this.$storageService.getItemById(id);
-      this.$emit("submit-session", this.activeSession);
+      console.log("handleActiveSession", { id });
+      this.itemToEdit = await this.$storageService.getItemById(id);
+      console.log("item to edit", this.itemToEdit);
+      this.editEvidenceDialog = true;
+
+      // this.activeSession = await this.$storageService.getItemById(id);
+      // this.$emit("submit-session", this.activeSession);
```
Comment (dacoaster) [inline src/components/TimelineWrapper.vue:1144]: @isntlazy - Please remove unused lines.

File: src/modules/DatabaseUtility.js [hunk 3]
```diff
@@ -262,6 +262,7 @@ const removeItemById = (id) => {
 
 const getItemById = (id) => {
   const data = dataDb.get("items");
+  console.log({ dataDb });
```
Comment (dacoaster) [inline src/modules/DatabaseUtility.js:265]: @isntlazy - We probably don't want to log the entire dataDb here.

File: src/components/TimelineWrapper.vue (hunk 1 above)
Comment (isntlazy) [reply src/components/TimelineWrapper.vue:1081]: sure

File: src/components/TimelineWrapper.vue (hunk 2 above)
Comment (isntlazy) [reply src/components/TimelineWrapper.vue:1144]: sure

File: src/modules/DatabaseUtility.js (hunk 3 above)
Comment (isntlazy) [reply src/modules/DatabaseUtility.js:265]: sure

File: src/views/MainView.vue [hunk 4]
```diff
@@ -145,10 +145,13 @@ export default {
...
     };
   },
+  created() {
+    this.fetchItems();
+  },
   mounted() {
     this.setInitialPreSession();
     this.setInitialPostSession();
-    this.fetchItems();
+    // this.fetchItems();
```
[the remark this answers is not in the document]
Comment (isntlazy) [reply src/views/MainView.vue:154]: sure

**Comment by isntlazy** [comment]
@dacoaster I cleared all of this here: https://github.com/yatt-ai/yattie/pull/116

**Pull request merged**